shell/machine-config/azure.vue

```diff
@@ -209,7 +230,7 @@ export default {
           :searchable="false"
           :required="true"
           :label="t('cluster.machineConfig.azure.environment.label')"
-          :disabled="disabled"
+          :disabled="true"
```

Comment:
Since this select will always be disabled, would it make sense change this to the LabeledInput component and remove `azureEnvironments` from this config?

Reply:
I removed azureEnvironments and changed it to the LabeledInput component. In addition, for supporting multiple environments, this function depends on back-end code. I found that the back-end code has not been merged.

Reply:
With that in mind, this should be merged once the back-end code is in.